### ISSUE TITLE
[rst/en] double backquote example is unbalanced

### ISSUE THREAD

**Issue by Qix-** [adambard/learnxinyminutes-docs#3267]
```
You can  put text in *italic* or in **bold**, you can "mark" text as code with double backquote ``: ``print()``.
```

This makes no sense. There are three double backquotes there. Is there an error with how it's rendered?

<img width="913" alt="image" src="https://user-images.githubusercontent.com/885648/46604502-b6d26a80-cab3-11e8-8d3a-22986ad760c2.png">

Or do you prefix every token in code with double-backquotes? But that wouldn't make sense either because `print()` is suffixed with double backquotes.

**Comment by divayprakash** [comment]
@Qix- Thanks for noticing that!

Closed in 215d688.

**Comment by Qix-** [comment]
Thanks, that's much clearer :)
